metaseq/data/jsonl_dataset.py

```diff
@@ -88,6 +88,13 @@ def _build_index(self, path: str):
         cur = 0
         while True:
             line = f.readline()
+            if line != b"":
```

Comment:
why is this bytes?

Reply:
I noticed that that's just how line was formatted. And if line was b"" then it would always fail json loading anyway